narwhals/expr.py

```diff
@@ -1494,14 +1494,16 @@ def sample(
         )
 
     def over(
-        self: Self, *keys: str | Iterable[str], _order_by: str | None = None
+        self: Self,
+        *partition_by: str | Iterable[str],
```

Comment:
Any reason to not align with:

```py
partition_by: str | Iterable[str],
*more_exprs: str,
```

at the Narwhals level only?
I think the main difference in being able to pass `partition_by` as keyword argument.

Reply:
like this it's a bit more natural not pass any `partition_by`, whereas the Polars way it's required

Reply:
So are we planning to default it to None eventually to allow `.max().over()`?

Reply:
the signature from this PR allows that, if you don't pass any positional arguments then it's an empty over

Reply:
🤦🏼‍♂️